modules/drivers/st7789/st7789.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2017  Moddable Tech, Inc.
+ *
+ *   This file is part of the Moddable SDK Runtime.
+ * 
+ *   The Moddable SDK Runtime is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Lesser General Public License as published by
+ *   the Free Software Foundation, either version 3 of the License, or
+ *   (at your option) any later version.
+ * 
+ *   The Moddable SDK Runtime is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU Lesser General Public License for more details.
+ * 
+ *   You should have received a copy of the GNU Lesser General Public License
+ *   along with the Moddable SDK Runtime.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/*
+	ST7789VW (Sitronix) SPI display controller
+*/
+
+import ILI9341 from "ili9341";
+
+export default class ST7789 extends ILI9341 {};
```

Comment:
Since this ST7789 is just an alias for ILI9341, I think you can simplify this to:

```js
export default ILI9341;
```

That avoids an empty link in the instance's prototype chain.